benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -249,7 +249,7 @@ private List<QueryExecutionResult> executeConcurrentQueries(Benchmark benchmark,
                         firstQuery = false;
```

Comment:
Please update your commit description to describe the problem scenario and the fix
E.g.
```
Benchmark results were not saved in benchto-service in XYZ scenario.
This is resolved by doing ABC.
```